### PR TITLE
Standardise ug

### PR DESCRIPTION
**Pull request by junwei-tan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -213,41 +213,56 @@ Example:
...
 
 * Contacts matching at least one keyword will be returned. e.g. `n/Perry Dr`
   will match contacts with name `Perry the Platypus` and `Dr Doofenshmirtz`.
 
-Examples:
+<div markdown="block" class="alert alert-warning">
+
+:warning: There will be an error if you:<br>
+
+* Do not adhere to the [constraints](#section-1-contacts) for each field.<br>
```
Comment (feliciagan) [inline docs/UserGuide.md]: Change to Do not adhere to the field constraints to standardise

File: docs/UserGuide.md [hunk 2]
```diff
@@ -213,41 +213,56 @@ Example:
...
 
-* `findC n/flynn` will return `Candace Flynn` and `Phineas Flynn`.
+* `findC n/john p/81113210 e/john@gmail.com a/123` will return `Candace Flynn` and `Phineas Flynn`.
 
+* `findC n/john p/12345678 e/john@gmail.com a/123` will return contacts with name containing the word `john`, phone number `12345678`, email `john@gmail.com`, and address containing `123`.
 
 ### Filtering contacts by label: `filterC`
 
-Filters contacts whose label(s) contain any of the given keywords.
+Filters contact whose label(s) contain any of the given keywords.
```
Comment (feliciagan) [inline docs/UserGuide.md]: Should it not be plural?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -213,41 +213,56 @@ Example:
...
 
-Finds a contact using one or more information fields (e.g. name, mobile number, email, address, and/or remark)
+Finds a contact using one or more information fields (e.g. name, phone number, email, address, and/or remark).
 
 Format: `findC [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [r/REMARK]`
 
-* At least one information field has to be provided.
-
-* The search is case-insensitive, e.g. `dr doofenshmirtz` will match `Dr Doofenshmirtz`.
+* The search is case-insensitive. e.g. `dr doofenshmirtz` will match `Dr Doofenshmirtz`.
```
Comment (feliciagan) [inline docs/UserGuide.md]: leave as comma (case-insensitive,)

File: docs/UserGuide.md [hunk 4]
```diff
@@ -213,41 +213,56 @@ Example:
...
 
 * Only full words will be matched. e.g. `John` will not match `Johnny`.
```
Comment (feliciagan) [inline docs/UserGuide.md]: change to comma before e.g.

Comment (feliciagan) [reply docs/UserGuide.md]: do for all the e.g.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -213,41 +213,56 @@ Example:
...
 
-* `findC n/john p/81113210 e/john@gmail.com a/123` will return a contact whose name contains the word `john`, phone number `81113210`, email `john@gmail.com`, and address containing `123`.
+* Do not provide at least one information field.<br>
+
+</div>
+
+Examples:
 
-* `findC n/flynn` will return `Candace Flynn` and `Phineas Flynn`.
+* `findC n/john p/81113210 e/john@gmail.com a/123` will return `Candace Flynn` and `Phineas Flynn`.
```
Comment (feliciagan) [inline docs/UserGuide.md]: Is this supposed to be deleted?

File: docs/UserGuide.md [hunk 6]
```diff
@@ -213,41 +213,56 @@ Example:
...
 ### Filtering contacts by label: `filterC`
 
-Filters contacts whose label(s) contain any of the given keywords.
+Filters contact whose label(s) contain any of the given keywords.
 
-Format: `filterC KEYWORD [MORE_KEYWORDS]`
+Format: `filterC KEYWORD [MORE_KEYWORDS]...`
 
-* The filter is case-sensitive, e.g. `cs2103t` will not match `CS2103T`.
+* The filter is case-sensitive. e.g. `cs2103t` will not match `CS2103T`.
```
Comment (feliciagan) [inline docs/UserGuide.md]: change back to comma

File: docs/UserGuide.md [hunk 7]
```diff
@@ -213,41 +213,56 @@ Example:
...
 
-* Only the contact's label is filtered.
+* Only full words will be matched. e.g. `math` will not match `mathematics`.
```
Comment (feliciagan) [inline docs/UserGuide.md]: this too

File: docs/UserGuide.md [hunk 8]
```diff
@@ -213,41 +213,56 @@ Example:
...
 
-* Only full words will be matched. e.g. `cs2103t` will not match `cs2103`.
+<div markdown="block" class="alert alert-warning">
 
-Example:
+:warning: There will be an error if you:<br>
+
+* Do not adhere to the name constraint for labels. Label names must be alphanumeric with no spaces.
```
Comment (feliciagan) [inline docs/UserGuide.md]: Change to Do not adhere to label name constraints to standardise

File: docs/UserGuide.md [hunk 9]
```diff
@@ -386,25 +411,37 @@ Format: `findT [d/DESCRIPTION] [D/DEADLINE (dd-mm-yyyy)] [s/STATUS (complete / i
 * Task descriptions matching at least one keyword will be returned. e.g. `d/cs2103t cs2101` will match
   `cs2103t tutorial` and `cs2101 reflection`.
 
+<div markdown="block" class="alert alert-warning">
+
+:warning: There will be an error if you:<br>
+
+* Do not adhere to the [constraints](#section-2-tasks) for each field.<br>
```
Comment (feliciagan) [inline docs/UserGuide.md]: standardise

File: docs/UserGuide.md [hunk 10]
```diff
@@ -386,25 +411,37 @@ Format: `findT [d/DESCRIPTION] [D/DEADLINE (dd-mm-yyyy)] [s/STATUS (complete / i
...
+
+* Do not provide at least one information field.<br>
```
Comment (feliciagan) [inline docs/UserGuide.md]: Change to Do not provide at least one of the optional fields to standardise

File: docs/UserGuide.md [hunk 11]
```diff
@@ -213,41 +213,56 @@ Example:
...
 
-Examples:
+<div markdown="block" class="alert alert-warning">
+
+:warning: There will be an error if you:<br>
+
+* Do not adhere to the [constraints](#section-1-contacts) for each field.<br>
 
-* `findC n/john p/81113210 e/john@gmail.com a/123` will return a contact whose name contains the word `john`, phone number `81113210`, email `john@gmail.com`, and address containing `123`.
+* Do not provide at least one information field.<br>
```
Comment (feliciagan) [inline docs/UserGuide.md]: Change to Do not provide at least one of the optional fields to standardise

File: docs/UserGuide.md [hunk 12]
```diff
@@ -386,25 +411,37 @@ Format: `findT [d/DESCRIPTION] [D/DEADLINE (dd-mm-yyyy)] [s/STATUS (complete / i
...
 
 ### Filtering tasks by label: `filterT`
 
 Filters tasks whose label(s) contain any of the given keywords.
 
-Format: `filterT KEYWORD [MORE_KEYWORDS]`
+Format: `filterT KEYWORD [MORE_KEYWORDS]...`
 
-* The filter is case-sensitive, e.g. `cs2103t` will not match `CS2103T`.
+* The filter is case-sensitive. e.g. `cs2103t` will not match `CS2103T`.
```
Comment (feliciagan) [inline docs/UserGuide.md]: change back to comma

File: docs/UserGuide.md [hunk 13]
```diff
@@ -386,25 +411,37 @@ Format: `findT [d/DESCRIPTION] [D/DEADLINE (dd-mm-yyyy)] [s/STATUS (complete / i
...
 
-* Only full words will be matched. e.g. `cs2103t` will not match `cs2103`.
+* Only full words will be matched. e.g. `math` will not match `mathematics`.
```
Comment (feliciagan) [inline docs/UserGuide.md]: change to comma before eg

File: docs/UserGuide.md [hunk 14]
```diff
@@ -314,22 +333,34 @@ Shows all (non-archived) tasks stored in the task list.
 
 Format: `listT`
 
-* Tasks are marked as unarchived when they are first added by default.
+* Newly added tasks are marked as unarchived by default.
+
+<div markdown="span" class="alert alert-info">
```
Comment (alvintfl) [inline docs/UserGuide.md]: Shouldn't the markdown="block"?

File: docs/UserGuide.md [hunk 15]
```diff
@@ -314,22 +333,34 @@ Shows all (non-archived) tasks stored in the task list.
...
+
 ### Listing all archived tasks: `listAT`
 
-Shows all archived tasks stored in the task list. Tasks can be unarchived
+Shows all archived tasks stored in the task list.
 
 Format: `listAT`
 
-* Unarchived tasks can be viewed using the [`listAT`](#listing-all-non-archived-tasks-listt) command.
+<div markdown="span" class="alert alert-info">
```
Comment (alvintfl) [inline docs/UserGuide.md]: Shouldn't the markdown="block"?

**Comment by alvintfl** [comment]
LGTM

**Pull request merged**